src/database/models/wishlistModel.ts

```diff
@@ -0,0 +1,60 @@
+import { DataTypes, Model } from 'sequelize'
+import { v4 as uuidv4 } from 'uuid'
+import sequelizeConnection from '../config/db.config'
+import User from './userModel'
+import Product from './productModel'
+
+/**
+ * Wishlist Interface
+ */
+interface WishlistAttributes {
+  id?: string
+  buyerId: number
+  productId: string
+}
+
+/**
+ * Represents a wishlist in the database.
+ */
+class Wishlist extends Model<WishlistAttributes> implements WishlistAttributes {
+  public id!: string
+
+  public buyerId!: number
+
+  public productId!: string
+
+  /**
+   * Initializes the Wishlist model.
+   * @param {object} sequelize The Sequelize instance.
+   * @returns {void}
+   */
+  static associate(): void {
+    Wishlist.belongsTo(User, { foreignKey: 'buyerId' })
```

Comment:
```suggestion
    Wishlist.belongsTo(User, { foreignKey: 'buyerId' })
```
did you remember to add the relation to user too, to show that user has one or many wishlist?